src/data/polynomial.lean

```diff
@@ -2034,8 +2160,62 @@ lemma X_pow_sub_C_ne_zero {n : ℕ} (hn : 0 < n) (a : R) :
 mt degree_eq_bot.2 (show degree ((X : polynomial R) ^ n - C a) ≠ ⊥,
   by rw degree_X_pow_sub_C hn a; exact dec_trivial)
 
+lemma nat_degree_X_sub_monomial_zero {r : R} : (X - monomial 0 r).nat_degree = 1 :=
```

Comment:
Idem dito... and more below
Should we have a simp lemma that changes `monomial 0 r` into `C r`?